tests/test_querysetsequence.py

```diff
@@ -340,6 +340,14 @@ def test_prefetch_related(self):
             authors = [b.author.id for b in books]
         self.assertEqual(authors, self.EXPECTED_ORDER)
 
+    def test_prefetch_related_lookups(self):
```

Comment:
Please add a docstring about what this is testing.

Reply:
Would it make more sense to create the `ModelChoiceField` and call the API on it to ensure it "works" as expected?

Reply:
I added this test as suggested.